src/mcp_atlassian/servers/main.py

```diff
@@ -532,13 +538,16 @@ async def _send_error_response(
 
 
 main_mcp = AtlassianMCP(name="Atlassian MCP", lifespan=main_lifespan)
-main_mcp.mount("jira", jira_mcp)
-main_mcp.mount("confluence", confluence_mcp)
+main_mcp.mount(jira_mcp, prefix="jira")
+main_mcp.mount(confluence_mcp, prefix="confluence")
 
 
 @main_mcp.custom_route("/healthz", methods=["GET"], include_in_schema=False)
 async def _health_check_route(request: Request) -> JSONResponse:
     return await health_check(request)
 
 
-logger.info("Added /healthz endpoint for Kubernetes probes")
+@main_mcp.custom_route("/ready", methods=["GET"], include_in_schema=False)
```

Comment:
```suggestion
@main_mcp.custom_route("/readyz", methods=["GET"], include_in_schema=False)
```

If `healthz` is already in use we should use `readyz` for consistency - https://kubernetes.io/docs/reference/using-api/health-checks/

Reply:
makes sense. Let me fix that